app/uk/gov/hmrc/voapropertylinking/connectors/modernised/OrganisationManagementApi.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.voapropertylinking.connectors.modernised
+
+import javax.inject.{Inject, Named}
+import uk.gov.hmrc.http._
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import uk.gov.hmrc.voapropertylinking.auth.RequestWithPrincipal
+import uk.gov.hmrc.voapropertylinking.http.VoaHttpClient
+import uk.gov.hmrc.voapropertylinking.models.modernised.agentrepresentation.{AgentDetails, AppointmentChangeResponse, AppointmentChangesRequest}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class OrganisationManagementApi @Inject()(
```

Comment:
ExternalOrganisationManagementApi so as to not confuse with the other API.

Reply:
i didn't know we had an "organisation-management-api" but I have now changed the name to ExternalOrganisationManagementApi